### PR TITLE
Badge button documentation

### PR DESCRIPTION
**Pull request by Aazam10**
Hi I have added documentation for badge and button component.
CSB FOR BUTTON:https://lluo6.csb.app/button.html
CSB FOR BADGE: https://mnll3.csb.app/badge.html

Reviews are appreciated. Thanks

### CODE REVIEW

**Review by Husain01 — APPROVED** [review]
Looks good to go

File: Button/button.html [hunk 1]
```diff
@@ -6,33 +6,221 @@
...
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </div>
+          <div class="doc-line"></div>
+
+          <!-- ICON BUTTON -->
+          <div class="doc-content-title">Icon Button</div>
+          <div class="doc-content-desc">
+            Used when you eant to show icons with the text of the button.
```
Comment (mohammed-sas) [inline Button/button.html]: typo : eant --> want

**Review by mohammed-sas — APPROVED** [review]
pr good to merge

File: Button/button.html (hunk 1 above)
Comment (Aazam10) [reply Button/button.html]: done

**Pull request merged**